.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  release:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+        architecture: x64
+    - name: Build with Gradle
+      env:
+        JAVA_OPTS: -Xms512m -Xmx1024m
+      run: |
+        ./gradlew clean build
+    - name: Publish artifacts
+      env:
+        BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
+        BINTRAY_API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+      run: |
+        ./gradlew uploadArchives
```

Comment:
Are we not using `./deploy-scripts/deploy.sh` anymore like we did for Travis?

Reply:
Thanks for the review @nomisRev :+1: We simplified that part in Arrow Meta when including what those scripts were doing. In this way, we consider that it's easier to know what's happening in GitHub Actions. Thanks!

Reply:
Typing on phone... Opening & closing... Sorry :D

Awesome! That's what I was hoping for.

Thanks for the update.

Reply:
Nothing to apologize, reviews are always useful, mate!! Thanks to you!! :raised_hands: